lib/Cargo.toml

```diff
@@ -188,6 +188,8 @@ yara-x-parser = { workspace = true }
 yara-x-proto = { workspace = true }
 
 lingua = { version = "1.6.0", optional = true, default-features = false, features = ["english", "german", "french", "spanish"] }
+roxmltree = "0.19.0"
```

Comment:
Both `roxmltree` and `cryptographic-message-syntax` crates are only used by the `macho` module, they should be made an optional dependency that is included only if the `macho-module` feature is enabled.

Reply:
Fixed i believe 😸